uvicorn/config.py

```diff
@@ -370,6 +370,7 @@ def __init__(
         if workers is None and "WEB_CONCURRENCY" in os.environ:
             self.workers = int(os.environ["WEB_CONCURRENCY"])
 
+        self.forwarded_allow_ips: Union[List[str], str]  # help mypy
```

Comment:
What happens if this is removed?

Reply:
then mypy gets confused about the assignments below.

```
uvicorn/config.py:378: error: Incompatible types in assignment (expression has type "Union[List[str], str]", variable has type "str")  [assignment]
```

should I clarify the comment a bit more? :)